cli/src/main/resources/nls/Help.properties

```diff
@@ -89,7 +89,7 @@ cmd.set-version.detail=This will set the according tool version variable in your
 cmd.set-version.opt.--cfg=Selection of the configuration file (settings | home | conf | workspace).
 cmd.set-version.val.version=The tool version to set.
 cmd.shell=Commandlet to start built-in shell with advanced auto-completion.
-cmd.shell.detail=TODO shell
+cmd.shell.detail=The interactive shell offers console users a new user experience with an advanced auto-completion. Most inputs can be autocompleted in this mode.\nIDEasy will offer several matching commands e.g. 'install' or options e.g. '-t' by simply hitting the 'Tab' key after the expression f.e. 'in' for 'install' and 'intellij'. You can use the Tab key to switch between both suggestions.\nTo exit this mode use 'CTRL+C', 'CTRL+D' or type 'exit'.
```

Comment:
Great explanation!

```
IDEasy will offer several matching commands e.g. 'install' or options e.g. '-t' by simply hitting the 'Tab' key after the expression f.e. 'in' for 'install' and 'intellij'. You can use the Tab key to switch between both suggestions.
```

Isn't that the same in git-bash completion?
Maybe here we should rather describe the unique features added on top like cursor navigation within the suggested options, etc.

Reply:
I've adjusted the explanation and added a git bash and jline3 reference to it now.